projects/js-themes-toolkit/packages/liferay-theme-tasks/theme/tasks/build/index.js

```diff
@@ -34,7 +34,7 @@ function injectJS() {
 		'/liferay-frontend-theme-unstyled/templates/portal_normal\\.(ftl|vm)$'
 	);
 
-	return through.obj(function (file, encoding, callback) {
+	return through.object(function (file, encoding, callback) {
```

Comment:
Same issue here. We should make sure the identifier we are checking is not a key on an object